src/components/Attachments/AttachmentView/index.tsx

```diff
@@ -117,7 +116,7 @@ function AttachmentView({
         if (!isFocused && !(file && isUsedInAttachmentModal)) {
             return;
         }
-        updateCurrentlyPlayingURL(isVideo ? source : null);
+        updateCurrentlyPlayingURL(isVideo ? (source as string) : null);
```

Comment:
```suggestion
        updateCurrentlyPlayingURL(isVideo && typeof source === 'string' ? source : null);
```

Avoid type assertion.